.github/pull_request_template.md

```diff
@@ -2,11 +2,11 @@
 What does this PR do?
 
 ### TODO only if you're a HashiCorp employee
-- [ ] **Labels:** If this PR is the CE portion of an ENT change, and that ENT change is
-  getting backported to N-2, use the new style `backport/ent/x.x.x+ent` labels
-  instead of the old style `backport/x.x.x` labels.
-- [ ] **Labels:** If this PR is a CE only change, it can only be backported to N, so use
-  the normal `backport/x.x.x` label (there should be only 1).
+- [ ] **CE Labels:** If this PR is a CE-only change, you should only backport to N, so use
```

Comment:
Yeah I added N to the enterprise labels section because it wasn’t there before and I found that misleading too. 

But I think you’re right that we can simplify at least the Enterprise and CE checkboxes into one instead of cognitively overloading people with too many checkboxes; I was perhaps needlessly trying to maintain some of the original checklist items’ wording. 

However I think LTS does need to be called out as an explicit item because I think it will be easy to “go through the motions” and apply the usual backlog labels without remembering the LTS branches, which are not being updated with every ent change, only the critical ones.